src/components/strategies/edit/EditStrategyBudgetContent.tsx

```diff
@@ -172,16 +184,19 @@ export const EditStrategyBudgetContent = ({
       />
       <Button
         disabled={!isOrdersBudgetValid()}
+        loading={isLoading}
+        loadingChildren={loadingChildren}
         onClick={handleOnActionClick}
         className="mt-32"
-        variant="white"
+        variant={'white'}
         size="lg"
         fullWidth
       >
         {type === 'withdraw' ? 'Confirm Withdraw' : 'Confirm Deposit'}
       </Button>
       <Button
         onClick={() => back()}
+        disabled={isLoading}
```

Comment:
do we wanna disable the cancel button when loading?

Reply:
Yes, we wanna disable both cancel button and call to action button.